app/services/service_provider_request_proxy.rb

```diff
@@ -4,9 +4,10 @@
 # To migrate code simply replace ServiceProviderRequest with ServiceProviderRequestProxy
 class ServiceProviderRequestProxy
   REDIS_KEY_PREFIX = 'spr:'.freeze
-  REDIS_LAST_UUID_KEY = 'spr_last_uuid'.freeze
   DEFAULT_TTL_HOURS = 24
 
+  cattr_accessor :redis_last_uuid
```

Comment:
can we add a comment here that it's only for tests (I know `.last` has a comment too)

Reply:
Yeah, will do before I merge